sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAdminClient.java

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.personalizer;
+
+import com.azure.ai.personalizer.implementation.models.*;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.StreamResponse;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Flux;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+
+public final class PersonalizerAdminClient {
+
+    private PersonalizerAdminAsyncClient client;
+
+    public PersonalizerAdminClient(PersonalizerAdminAsyncClient client) {
+        this.client = client;
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Evaluation createEvaluation(EvaluationContract evaluationContract) {
+        return createEvaluationWithResponse(evaluationContract, Context.NONE).getValue();
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Evaluation> createEvaluationWithResponse(EvaluationContract evaluationContract, Context context) {
+        return client.createEvaluationWithResponse(evaluationContract, context).block();
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Evaluation getEvaluation(String evaluationId) {
+        return getEvaluationWithResponse(evaluationId, Context.NONE).getValue();
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Evaluation> getEvaluationWithResponse(String evaluationId, Context context) {
+        return client.getEvaluationWithResponse(evaluationId, context).block();
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void deleteEvaluation(String evaluationId) {
+        deleteEvaluationWithResponse(evaluationId, Context.NONE).getValue();
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Void> deleteEvaluationWithResponse(String evaluationId, Context context) {
+        return client.deleteEvaluationWithResponse(evaluationId, context).block();
+    }
+
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public List<Evaluation> getEvaluations() {
+        return getEvaluationsWithResponse(Context.NONE).getValue();
+    }
+
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public Response<List<Evaluation>> getEvaluationsWithResponse(Context context) {
```

Comment:
Is this not a paged API?

Reply:
No. Currently, preview api version does not have paging. We will add support for this in the next version.